daffodil-lib/src/main/scala/org/apache/daffodil/lib/util/Misc.scala

```diff
@@ -708,4 +708,19 @@ object Misc {
     }
     Some(res)
   }
+
+  /**
+   * Strip the part of the path matching the classpath from the uri
+   * @param uriString
+   * @return
+   */
+  def stripJarPrefix(uriString: String): String = {
+    val withoutJar = if (!isNullOrBlank(uriString)) {
+      uriString.split("\\.jar!").last
+    } else {
+      uriString
+    }
+    withoutJar
+  }
```

Comment:
In windows, we ran into some places where a jar uri had a "file" scheme, so I'm going to treat both schemes the same

Reply:
So the URI was something like `file://path/to/foo.jar!/foo.dfdl.xsd` but with windows paths? I don't think that is a valid file path, since anything opening that is going to look for a file called `foo.dfdl.xsd" in a directory called `foo.jar!`, I think. I wonder if we had a bug somewhere else that lead to that invalid file scheme URI?

Reply:
Ah I see. I'll treat them differently and reinvestigate why it does that in Windows